Documentation/ColumnsConfig/Type/Link/Index.rst

```diff
@@ -102,6 +102,26 @@ where code adoption has to take place.
    being stored in the database. Therefore, the :php:`eval=trim` option is no
    longer needed and should be removed from the TCA configuration.
 
+Create an URL
+=============
+
+To create a URL from such a link field in a Fluid template, use the
+`<f:link.typolink>` or `<f:uri.typolink>` view helper.
+
+In PHP code, use `ContentObjectRenderer::createUrl`:
```

Comment:
I am currently not sure if the ContentObjectRenderer is the right choice (although it might work). Will have a look.

Reply:
fixed.